src/MagnumPlugins/KtxImageConverter/KtxImageConverter.cpp

```diff
@@ -729,7 +729,7 @@ Containers::Array<char> KtxImageConverter::convertLevels(Containers::ArrayView<c
             return {};
         }
 
-        constexpr Containers::StringView validOrientations[3]{"rl"_s, "du"_s, "io"_s};
```

Comment:
Any idea what's happening here? GCC 4.8 kept complaining about:

> error: cannot initialize aggregate of type 'const StringView [3] {aka const Corrade::Containers::BasicStringView<const char> [3]}' with a compound literal

but the **exact** same code works perfectly fine in `KtxImporter::doOpenData`.

Reply:
Heh. I suppose it's because the function is templated? Putting it outside (and into a `namespace {`) should make the `constexpr` work, I hope.